lib/src/layer/tile_layer.dart

```diff
@@ -53,6 +53,9 @@ class TileLayerOptions extends LayerOptions {
   ///Color shown behind the tiles.
   final Color backgroundColor;
 
+  ///Container Widget decoration
+  final Decoration decoration;
```

Comment:
Why is this required? If you wanted decoration around the map, couldn't the map widget be wrapped in a Container?

Reply:
I added it to test some styling but you are right, its probably unnecessary.